conan/tools/files/files.py

```diff
@@ -1,26 +1,28 @@
 import configparser
 import errno
+import gzip
+import hashlib
 import os
 import platform
 import subprocess
+import sys
 from contextlib import contextmanager
+from fnmatch import fnmatch
 
 from conan.tools import CONAN_TOOLCHAIN_ARGS_FILE, CONAN_TOOLCHAIN_ARGS_SECTION
 from conans.client.downloaders.download import run_downloader
-from conans.client.tools.files import unzip, which
+from conans.client.tools.files import which
```

Comment:
which in Conan 2.0 is going to be just ``shutil.which()``. Do we still want a wrapper? (Maybe necessary for Py2 only?)

Reply:
I assume, the new tools at "conan" will be python3 (because conan2 is python3) so I'll remove the import (I'll keep the implementation to not break) and use `shutil.which()` here.